readme.rst

```diff
@@ -12,7 +12,25 @@ work with data in the BAG format:
 * **api** - This is the primary API directory and contains the source for the Bathymetric Attributed Grid format (BAG).
 * **configdata** - Required XML support files. You must have an environment variable called BAG_HOME mapped to this directory in order to run the API functions.
 * **docs** - As the documentation evolves it will reside here. Currently this contains some initial draft documentation.
-* **examples** - Contains programs to demonstrate some of the API functionality. In particular bagcreate and bagread are good starting points.
-* **extlibs** - Contains the 3rd-party libraries upon which the API depends. The examples will also need to link with the libraries in this directory in order to build properly.
+* **examples** - Contains programs to demonstrate some of the API functionality. In particular bag_create and bag_read are good starting points.
+* **python** - Contains Python tests that make use of the SWIG interface.
+* **tests** - Contains C++ tests.
 
 *Important Building Notes*: The build instructions can be found here: docs/build_instructions.html
+
+Building documentation
+----------------------
+Doxygen is needed to produce the documentation.  Get it at http://www.doxygen.nl/
+
+::
+    cd docs/api
+    doxygen docs_config.dox
+
+  Browse to html/index.html
+Running C++ tests in Visual Studio
+----------------------------------
+The C++ unit tests are written with Catch2.  There is no test harness at the moment.  Using the Standard toolbar, choose bag_tests.exe as the Select Startup Item.  Press F5 to run the tests.
```

Comment:
Do you mean the 'Solution Explorer'? I don't understand how to use the 'Standard toolbar' for this task.
Did you forget the instructions about setting the '--samples-path'?